### ISSUE TITLE
Key error and division by zero

### ISSUE THREAD

**Issue by taylorreiter** [metagenome-atlas/atlas#204]
```
ubuntu@ip-172-31-23-147:/mnt/work$ snakemake --snakefile /mnt/work/atlas/atlas/Snakefile --directory /mnt/work --printshellcmds --jobs 1 --rerun-incomplete --configfile '/mnt/work/config.yaml' --nolock  --use-conda  --conda-prefix /mnt/work/databases/conda_envs   genomes
Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1
Rules claiming more threads will be scaled down.
Unlimited resources: mem, java_mem
Job counts:
        count   jobs
        1       align_reads_to_MAGs
        1       bam_2_sam_MAGs
        1       build_db_genomes
        1       cat_get_name
        1       combine_bined_coverages_MAGs
        1       combine_coverages_MAGs
        1       convert_sam_to_bam
        1       fasttree
        1       genomes
        1       merge_taxonomy
        1       parse_cat_output
        1       pileup_MAGs
        1       predict_genes_genomes
        1       rename_genomes
        1       root_tree
        1       run_all_checkm_lineage_wf
        1       run_checkm_tree_qa
        17

[Fri Apr 26 21:48:32 2019]
localcheckpoint rename_genomes:
    input: genomes/Dereplication/dereplicated_genomes, genomes/pre_dereplication/dereplicated_genomes
    output: genomes/genomes, genomes/clustering/contig2genome.tsv, genomes/clustering/old2newID.tsv, genomes/clustering/allbins2genome.tsv
    jobid: 6
Downstream jobs will be updated after completion.

Job counts:
        count   jobs
        1       rename_genomes
        1
/home/ubuntu/miniconda3/envs/atlas/bin/python3.6 /mnt/work/.snakemake/shadow/tmpf_ww9ydi/.snakemake/scripts/tmp6t509s0f.rename_genomes.py
Updating job 13.
Updating job 69.
Updating job 4.
Updating job 5.
Updating job 26.
Updating job 9.
Updating job 0.
[Fri Apr 26 21:48:35 2019]
Finished job 6.
[Fri Apr 26 21:48:35 2019]
Error in rule rename_genomes:
    jobid: 6
    output: genomes/genomes, genomes/clustering/contig2genome.tsv, genomes/clustering/old2newID.tsv, genomes/clustering/allbins2genome.tsv

Removing output files of failed job rename_genomes since they might be corrupted:
genomes/genomes, genomes/clustering/contig2genome.tsv, genomes/clustering/old2newID.tsv, genomes/clustering/allbins2genome.tsv
exception calling callback for <Future at 0x7fdea40d4550 state=finished returned NoneType>
Traceback (most recent call last):
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/site-packages/snakemake/executors.py", line 360, in _callback
    callback(job)
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/site-packages/snakemake/scheduler.py", line 338, in _proceed
    self.progress()
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/site-packages/snakemake/scheduler.py", line 477, in progress
    logger.progress(done=self.finished_jobs, total=len(self.dag))
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/site-packages/snakemake/logging.py", line 160, in progress
    self.handler(dict(level="progress", done=done, total=total))
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/site-packages/snakemake/logging.py", line 125, in handler
    handler(msg)
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/site-packages/snakemake/logging.py", line 316, in text_handler
    p = done / total
ZeroDivisionError: division by zero

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/concurrent/futures/_base.py", line 324, in _invoke_callbacks
    callback(self)
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/site-packages/snakemake/executors.py", line 371, in _callback
    error_callback(job)
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/site-packages/snakemake/scheduler.py", line 346, in _error
    self._handle_error(job)
  File "/home/ubuntu/miniconda3/envs/atlas/lib/python3.6/site-packages/snakemake/scheduler.py", line 356, in _handle_error
    self.running.remove(job)
KeyError: rename_genomes
```

**Comment by SilasK** [comment]
Could you try with latest snakemake version 5.4.5

**Comment by SilasK** [comment]
I assume this should be solved or be related to #217
